fbfmaproom/fbfmaproom-sample.yaml

```diff
@@ -1934,6 +1945,17 @@ countries:
                     colormap: precip
                     lower_is_worse: yes
                     format: number0
+                bad-years-v3:
```

Comment:
Same question about the version number here. In Trello, Dante explicitly said it should be called version 2.